src/satellite/test.js

```diff
@@ -988,10 +1005,14 @@ describe('Elastic()', () => {
   });
 
   describe('Tests for mock Elastic()', () => {
-    const client = Elastic();
-    const { mock } = client;
```

Comment:
This still, why do we need to change this?

Reply:
Ah, missed this, thank you.  @TueeNguyen please revert this bit.

Reply:
@rclee91, `afterEach()` calls `stop()` which has changed to close all Elasticsearch connections, so I want to open the connection again before each mock Elastic test